website/server/libs/user/index.js

```diff
@@ -239,4 +242,38 @@ export async function rebirth (req, res, { isV3 = false }) {
   await Promise.all(toSave);
 
   res.respond(200, ...rebirthRes);
-}
\ No newline at end of file
+}
+
+const bannedSlurRegexs = bannedSlurs.map((word) => new RegExp(`.*${word}.*`, 'i'));
+
+function usernameContainsSlur (username) {
```

Comment:
Sorry about this but this index.js file is already big enough, can you move the functions to a different file? like libs/user/username?